third_party/xla_client/pjrt_computation_client.cc

```diff
@@ -267,6 +267,7 @@ PjRtComputationClient::ExecuteComputation(
   xla::ExecuteOptions execute_options;
   execute_options.untuple_result = options.explode_tuple;
   execute_options.strict_shape_checking = false;
+
```

Comment:
is this one typo or intentation?

Reply:
It's intentional, to separate the options/parameters declaration and the usage in a function call.